docs/source/notes/custom.rst

```diff
@@ -34,19 +32,19 @@ set up the working files by
 
    from geneplexus import custom
 
-   custom.edgelist_to_node("path/to/net.edg", "path/to/data",
+   custom.edgelist_to_nodeorder("path/to/net.edg", "path/to/data",
                            "your_net_name")
 
-   # Set up adjacency feature
+   # Create adjacency feature representation
    custom.edgelist_to_matrix("path/to/net.edg", "path/to/data",
                              "your_net_name", "Adjacency")
 
-   # Alternatively, set up influence feature
+   # Create influence feature representation
```

Comment:
Should we keep ``Alternatively`` since the user should choose between the two?

Reply:
Actually, nevermind, this is fine - both can be generated but the user only needs to choose when executing geneplexus.